tests/sympc/protocol/aby3/aby3_test.py

```diff
@@ -152,7 +153,7 @@ def test_local_decomposition(get_clients, security_type):
 
     decompose = parallel_execution(ABY3.local_decomposition, session.parties)(args)
 
-    x1_sh, x2_sh, x3_sh = list(zip(*decompose))
+    x1_sh, x2_sh, x3_sh = zip(*map(lambda x: x[0], decompose))
```

Comment:
MAYBE: Can you use `starmap` for this? (if the expression would become cleaner)

Reply:
Sure

Reply:
I tried to think over it and tried to use starmap here George, It did not quite fit the scenario, here as do not to bitwise decompose and have only single share.

Reply:
or am I missing something?